docs/react-testing-library/setup.mdx

```diff
@@ -491,3 +491,72 @@ And register it using mocha's `-r` flag:
 ```
 mocha -r ./mocha-watch-cleanup-after-each.js
 ```
+
+### Vitest
+
+When using the react-testing-library with [Vitest](https://vitest.dev/),
+you need to install jsdom and the @testing-library:
+
+```bash npm2yarn
+npm install --save-dev vitest jsdom
+npm install --save-dev @testing-library/react @testing-library/jest-dom
+```
+
+Since the testing library uses jsdom, you must set the `environments`
```

Comment:
Neither `/react` nor `/dom` "use" JSDOM. But both libraries require **a** DOM implementation to be available. The config and recommendation for JSDOM is fine but the wording needs some tweaking to make sure people are aware that they could also use a Browser or other DOM implementations.

Reply:
Thank you for pointing this out. I tweak the wording to make it more clear.